Jetsurvey/app/src/main/res/values/themes.xml

```diff
@@ -17,7 +17,7 @@
 <resources>
     <!-- Base application theme. -->
```

Comment:
One last thing I noticed from the screenshots: the date picker (which I believe comes from the MDC-Android Views library) is using the M2 purple palette. We most likely need to change this XML theme parent to `Theme.Material3.DayNight.NoActionBar`

Reply:
Good catch! Yeah, that looks a lot better.